content/webapp/components/ExhibitionsAndEvents/ExhibitionsAndEvents.tsx

```diff
@@ -1,16 +1,17 @@
-import { UiExhibition } from '@weco/common/model/exhibitions';
+import { Exhibition } from '@weco/common/model/exhibitions';
 import { UiEvent } from '@weco/common/model/events';
 import { Link } from '@weco/common/model/link';
 import CardGrid from '../CardGrid/CardGrid';
+import { FunctionComponent } from 'react';
 
 type Props = {
-  exhibitions: UiExhibition[];
+  exhibitions: Exhibition[];
```

Comment:
These changes are because the new `transform()` method is returning an `Exhibition` instead of a `UiExhibition`. I strongly suspect the latter is redundant and could be safely deleted, but that'll be a separate patch.